sparse/impl/KokkosSparse_sptrsv_solve_impl.hpp

```diff
@@ -260,14 +327,14 @@ struct SptrsvWrap {
       // A = LU
       // A^-1 = U^-1 * L^-1
```

Comment:
The comments here are a bit confusing, especially the line below where you apply the matrix on the left side of `b` which is wrong.
I would say the following:
// Ax = LUx = Lz = b, we use the change of variable z = U*x
// z = L^-1 * b, first we solve for z, storing the result back into b
// x = U^-1 * z, second we solve for x, again storing the result back into b